### ISSUE TITLE
Unable to install - 404 errors when getting CRDs yaml files from github

### ISSUE THREAD

**Issue by paoloyx** [integr8ly/application-monitoring-operator#95]
Hi,

we're getting this error when trying to install the operator

```
error: unable to read URL "https://raw.githubusercontent.com/coreos/prometheus-operator/master/example/prometheus-operator-crd/podmonitor.crd.yaml", server reported 404 Not Found, status code=404
error: unable to read URL "https://raw.githubusercontent.com/coreos/prometheus-operator/master/example/prometheus-operator-crd/prometheus.crd.yaml", server reported 404 Not Found, status code=404
error: unable to read URL "https://raw.githubusercontent.com/coreos/prometheus-operator/master/example/prometheus-operator-crd/alertmanager.crd.yaml", server reported 404 Not Found, status code=404
error: unable to read URL "https://raw.githubusercontent.com/coreos/prometheus-operator/master/example/prometheus-operator-crd/prometheusrule.crd.yaml", server reported 404 Not Found, status code=404
error: unable to read URL "https://raw.githubusercontent.com/coreos/prometheus-operator/master/example/prometheus-operator-crd/servicemonitor.crd.yaml", server reported 404 Not Found, status code=404
```

**Comment by paoloyx** [comment]
Commit [ce72171](https://github.com/coreos/prometheus-operator/commit/ce72171c483921896d9370b3a8104c264fccdae7) on prometheus-operator seems to have changed referenced file paths

**Comment by paoloyx** [comment]
see PR #96

**Comment by paoloyx** [comment]
Won't fix until https://github.com/coreos/prometheus-operator/issues/2880 is fixed

**Comment by paoloyx** [comment]
[coreos/prometheus-operator#2880](https://github.com/coreos/prometheus-operator/issues/2880) has been closed, can someone review #96 in order to merge it? 

Thanks

**Comment by paoloyx** [comment]
Ok, issue has already been fixed by #101. Closing this one.
